scripts/install-server.sh

```diff
@@ -168,6 +168,13 @@ function check_trento_web_origin() {
     if [[ -z "$TRENTO_WEB_ORIGIN" ]]; then
         read -rp "A valid origin is required for websockets functionality, please provide one: " TRENTO_WEB_ORIGIN </dev/tty
     fi
+
+    if [[ -z "$TRENTO_WEB_ORIGIN" ]]; then
```

Comment:
@rtorrero I added these lines to make `TRENTO_WEB_ORIGIN` mandatory.
The script will continue ask you to provide it if you haven't done so.

Does it make sense?